ui/raidboss/data/triggers/o11s.js

```diff
@@ -1,13 +1,63 @@
+
 'use strict';
 
 // O11S - Alphascape 3.0 Savage
 [{
   zoneRegex: /^Alphascape V3.0 \(Savage\)$/,
   timelineFile: 'o11s.txt',
+  timelineTriggers: [
+    {
+      id: 'O11S Reset',
+      regex: /Reset/,
+      beforeSeconds: 6,
+      alertText: {
+        en: 'get out, center aoe',
```

Comment:
Personally I don't think there's any reason to callout obvious aoes on the ground that show up.